wordpress-mobile/WordPress-iOS
# Empty Filter Sheet View

Adds Empty View to Filter view from #13581

Original designs are here: https://wp.me/pauD4L-Dx

Related WordPress UI PR: https://github.com/wordpress-mobile/WordPressUI-iOS/pull/64

## To test
- Use a profile with no tags or filters or see instructions below in Testing
- Check that instructions to add a site or tag show up when no sites or tags are followed
- Check that Add a Site and Add a Tag button work.

## Screenshots

<img width="300" alt="Screen Shot 2020-04-23 at 2 53 48 PM" src="https://user-images.githubusercontent.com/3250/80148444-4c3c2f00-8572-11ea-9b64-0f96e0f76a21.png"><img width="300" alt="Screen Shot 2020-04-23 at 2 53 58 PM" src="https://user-images.githubusercontent.com/3250/80148454-4f371f80-8572-11ea-9cb8-ad33e8461ef3.png">

<details>
  <summary><strong>iPhone Screenshots</strong></summary>
<a href="https://user-images.githubusercontent.com/3250/80053876-2fa0e800-84db-11ea-8ef4-fd107b5927ea.png"><img src="https://user-images.githubusercontent.com/3250/80053876-2fa0e800-84db-11ea-8ef4-fd107b5927ea.png" width="600"></a>
<a href="https://user-images.githubusercontent.com/3250/80053948-57904b80-84db-11ea-872d-dbefb0150bb2.png"><img src="https://user-images.githubusercontent.com/3250/80053948-57904b80-84db-11ea-872d-dbefb0150bb2.png" width="300"></a><img width="300" alt="Screen Shot 2020-04-22 at 9 20 03 PM" src="https://user-images.githubusercontent.com/3250/80055518-0f732800-84df-11ea-9ad8-51579b1b16da.png">


</details>

<details>
  <summary><strong>iPad Screenshots</strong></summary>
<img width="500" alt="Screen Shot 2020-04-23 at 2 51 12 PM" src="https://user-images.githubusercontent.com/3250/80148227-f7001d80-8571-11ea-8ebc-b8d6574ba8f8.png">

</details>

#### Testing

You can manually remove the sites and tags items by returning an empty array from `FilterProvider`:

Replace `return items` with `return []` here:
https://github.com/wordpress-mobile/WordPress-iOS/blob/e76deb704bb62aa6b8fe181317144752abc10418/WordPress/Classes/ViewRelated/Reader/Filter/FilterProvider.swift#L35

PR submission checklist:
